activeresource.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.2'
 
-  s.add_dependency('activesupport', '~> 5.x')
-  s.add_dependency('activemodel',   '~> 5.x')
+  s.add_dependency('activesupport', '< 5.1')
```

Comment:
There are two different things here. Gemfile and gemspec. The gemspec defines the dependecies of the tem. In this case `;rails-observers', '~> 0.1.2'` is the actual dependency of this gem. Having it in the Gemfile of the gem doesn't make it a dependency.

You need to put the gem in your Gemfile because the rails-observers gem that support Rails 5 is not released yet, so you need to put it in your Gemfile otherwise it will not resolve to the right version. That is by the way the reason we have it in the activeresource Gemfile.

Could you change the version requirement to `'> 4.2, < 6'`?